### PR TITLE
Fixed lookup steps.cs and entity sub grid steps.cs

### PR DESCRIPTION
**Pull request by AhmedD43**
## Purpose
_Describe the problem or feature in addition to a link to the issue(s)._

## Approach
_How does this change address the problem?_

## TODOs
- [ ] Automated test coverage for new code
- [ ] Documentation updated (if required)
- [ ] Build and tests successful

### CODE REVIEW

**Comment by AhmedD43** [comment]
@tdashworth, here is the pull request relating to the fixes we have made on the specflow bindings project.

**Comment by osagiestar** [comment]
Looks good to me

File: bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LookupSteps.cs [hunk 1]
```diff
@@ -124,5 +124,20 @@ public static void WhenISelectARelatedLookupInTheForm(string lookupName)
...
+
+        public static void WhenIClickToPerformAnAdvancedLookup()
+
+        {
+            var fieldContainer = Driver.WaitUntilAvailable(By.Name("id-2015"));
+
+            fieldContainer.Click();
+
+            Driver.WaitForTransaction();
+        }
```
Comment (tdashworth) [inline bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LookupSteps.cs:141]: This change doesn't look related.

File: bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/EntitySubGridSteps.cs [hunk 2]
```diff
@@ -111,7 +111,11 @@ public static void ThenICanNotSeeInTheSubgrid(string alias, string subGridName)
         [Then(@"I can see (exactly|more than|less than) (\d+) records in the '(.*)' subgrid")]
         public static void ThenICanSeeRecordsInTheSubgrid(string compare, int count, string subGridName)
         {
-            var actualCount = XrmApp.Entity.SubGrid.GetSubGridItemsCount(subGridName);
+            //var actualCount = XrmApp.Entity.SubGrid.GetSubGridItemsCount(subGridName);
+            //Temporary until XrmApp.Entity.SubGrid.GetSubGridItemsCount(subGridName) works for a count of 0
```
Comment (tdashworth) [inline bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/EntitySubGridSteps.cs:115]: Please could we add a link to the EasyRepro issue.
```suggestion
            // Temporary until XrmApp.Entity.SubGrid.GetSubGridItemsCount(subGridName) works for a count of 0
            // https://github.com/microsoft/EasyRepro/issues/1318
```

**Review by tdashworth — CHANGES_REQUESTED** [review]
Thanks for the contribution 😄 a couple of comments if you could look at, please. 

Also, wasn't the same fix required for #137?

File: bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LookupSteps.cs (hunk 1 above)
Comment (osagiestar) [reply bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LookupSteps.cs:141]: We are currently reviewing that block of codes

**Comment by tdashworth** [comment]
**Manual validation requested**
Please perform any checks required on https://sfb-pr-8042.crm11.dynamics.com/ then approve https://dev.azure.com/capgeminiuk/GitHub%20Support/_build/results?buildId=8042.

**Comment by tdashworth** [comment]
**Manual validation requested**
Please perform any checks required on https://sfb-pr-8047.crm11.dynamics.com/ then approve https://dev.azure.com/capgeminiuk/GitHub%20Support/_build/results?buildId=8047.

**Comment by tdashworth** [comment]
After reviewing #137 again, it doesn't look to be the same solution.

File: bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LookupSteps.cs (hunk 1 above)
Comment (tdashworth) [reply bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LookupSteps.cs]: Resolved

**Comment by tdashworth** [comment]
**Manual validation requested**
Please perform any checks required on https://sfb-pr-8048.crm11.dynamics.com/ then approve https://dev.azure.com/capgeminiuk/GitHub%20Support/_build/results?buildId=8048.

**Review by tdashworth — APPROVED** [review]

**Comment by tdashworth** [comment]
Thanks for the contribution @AhmedD43 @osagiestar and team. 

When merging you can specify the commit message. Please follow to [convention commit pattern](https://www.conventionalcommits.org/en/v1.0.0/#summary) e.g. ‘fix: asserting sub-grid count when there are 0 actual rows’.

**Review by osagiestar — APPROVED** [review]
Looks good to me
